azure-toolkit-libs/azure-toolkit-mysql-lib/src/main/resources/com.microsoft.azure.toolkit.operation/titles_mysql.properties

```diff
@@ -0,0 +1,5 @@
+mysql|server.create=create MySQL server({})
+mysql|server.start=start MySQL server({})
+mysql|server.stop=stop MySQL server({})
+mysql|server.restart=restart MySQL server({})
+mysql|server.delete=delete MySQL server({})
```

Comment:
MySQL -> Azure Database for MySQL server

Reply:
No need, 
![image](https://user-images.githubusercontent.com/17075863/122322296-d6560c80-cf57-11eb-9906-9d2f837eb6b5.png)